scripts/biomedical/NCBI_Assembly/Summary_Report/scripts/process.py

```diff
@@ -0,0 +1,352 @@
+	Args:
+		df: dataframe with existing columns from source.
+	Returns:
+		df: dataframe with new columns with required terms.
+	
+	"""
+    df['dcid'] = 'bio/' + df['#assembly_accession']
+    df['genome_size_dcid'] = 'BasePairs' + df['genome_size'].astype(str)
+    df['genome_size_name'] = 'BasePairs ' + df['genome_size'].astype(str)
+    df['genome_size_ungapped_dcid'] = 'BasePairs' + df[
+        'genome_size_ungapped'].astype(str)
+    df['genome_size_ungapped_name'] = 'BasePairs ' + df[
+        'genome_size_ungapped'].astype(str)
+    df['gc_percent_dcid'] = 'Percent' + df['gc_percent'].astype(str)
+    df['gc_percent_name'] = 'Percent ' + df['gc_percent'].astype(str)
+    return df
+
+
+def refseq_category(df):
+    """Replacing columns values to required property value.
+	Args:
+		df: dataframe with existing columns values from source.
+	Returns:
+		df: dataframe with new columns values.
+	
+	"""
+    conversion_to_refseq_category = {
+        'representative genome': 'dcs:RefSeqCategoryRepresentativeGenome',
+        'reference genome': 'dcs:RefSeqCategoryReferenceGenome',
+    }
+    df['refseq_category'] = df['refseq_category'].map(
+        conversion_to_refseq_category).fillna(df['refseq_category'])
+    return df
+
+
+def tax_id(df):
+    """Replacing taxid column value as required.
+	Args:
+		df: dataframe with existing taxid column values from source.
+	Returns:
+		df: dataframe with required taxid column values.
+	
+	"""
+    df['taxid'] = df.apply(lambda row: '' if str(row['taxid']) == str(row[
+        'species_taxid']) else TAX_ID_DCID_MAPPING.get(row['taxid'], ''),
+                            axis=1)
+    return df
+
+
+def species_tax_id(df):
+    """Replacing species_taxid column value with tax_id_dcid_mapping.txt.
+	Args:
+		df: dataframe with existing species_taxid column values from source.
+	Returns:
+		df: dataframe with updated species_taxid column values.
+	
+	"""
+    df['species_taxid'] = df['species_taxid'].map(TAX_ID_DCID_MAPPING).fillna(
+        df['species_taxid'])
+    return df
+
+
+def infraspecific_name(df):
+    """Replacing unwanted char from infraspecific_name and organism_name column value.
+	Args:
+		df: dataframe with unwanted char in infraspecific_name and organism_name column values from source.
+	Returns:
+		df: dataframe with removed unwanted char.
+	
+	"""
+    df['infraspecific_name'] = df['infraspecific_name'].str.replace('/', ', ')
+    df['organism_name'] = df['organism_name'].str.replace('[', '').str.replace(
+        ']', '')
+    return df
+
+
+def format_correction(df):
+    """Replacing unwanted char from infraspecific_name and isolate column value.
+	Args:
+		df: dataframe with unwanted char in infraspecific_name and isolate column values from source.
+	Returns:
+		df: dataframe with removed unwanted char.
+	
+	"""
+    df['infraspecific_name'] = '\"' + df['infraspecific_name'] + '\"'
+    df['isolate'] = '\"' + df['isolate'] + '\"'
```

Comment:
Something inappropriate is happening with the strain values. First please strip the "strain=" from the input text value. More concerning, is that there is a formatting mishap that should not be occurring. For example, the input value for the isolate is "strain=BGD/2005/B109/[P67/H9/F67]" for "GCA_006446075.1". However, in the output file this is inappropriate converted to "strain=BGD, 2005, B109, [P67, H9, F67]". The actual value should be "GD/2005/B109/[P67/H9/F67]" in the output file. Please update the script and rerun to resolve this.